e2e/v1/language/testdata/rlang/build.envd

```diff
@@ -0,0 +1,6 @@
+# syntax=v1
+
+
+def build():
+    base(dev=True)
+    install.r_lang()
```

Comment:
Can you add a `install.r_packages()`?

Reply:
Just added it. However, it will fail when doing the e2e test as the other pull request is not merged. It will still try to install to `/usr/local/lib/R/site-library` as `envd` not `root`

Reply:
@kemingy

Reply:
merged that one, you can rebase the main